### PR TITLE
fix: Minting more than 1 NFT on the drop - gives you all random NFTs?

### LINKED ISSUE #9981: Minitng more than 1 NFT on the drop - gives you all random NFTs?
I was playing with variations, and I found the one I liked after minting - I selected two NFTs, and it showed me two different NFTs. At least one should be the one I generated, right? Or are they all random always?

https://github.com/kodadot/nft-gallery/assets/104380776/750c7546-888a-4b7e-b339-0556e5307bd4

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #9981

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

### CODE REVIEW

File: composables/drop/massmint/useDropMassMint.ts [hunk 1]
```diff
@@ -90,12 +90,17 @@ export default () => {
...
-            mintedAmountForCurrentUser.value,
-          )
+      const previewItems = (
+        single
+          ? [previewItem.value]
+          : [
+              previewItem.value,
+              ...generateMassPreview(
+                amountToMint.value - 1,
+                mintedAmountForCurrentUser.value,
```
Comment (hassnian) [inline composables/drop/massmint/useDropMassMint.ts:100]: CleanShot 2024-04-05 at 09 31 43@2x

untested but a `+ 1` should make it work correctly :d 

```suggestion
                mintedAmountForCurrentUser.value + 1,
```

**Review by hassnian — CHANGES_REQUESTED** [review]
entropy ranges should be unique.

File: composables/drop/massmint/useDropMassMint.ts (hunk 1 above)
Comment (Jarsen136) [reply composables/drop/massmint/useDropMassMint.ts:100]: Thanks!

**Review by hassnian — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by prury** [comment]
pay 35 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $35 @ 8.28 USD/DOT ~ 4.227 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x77d8ef8578e3f16a2a1a00ff365457565ab5bdf8088beae5d24e7456b3008e3d](https://polkadot.subscan.io/extrinsic/0x77d8ef8578e3f16a2a1a00ff365457565ab5bdf8088beae5d24e7456b3008e3d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**